docs/OV_Runtime_UG/model_representation.md

```diff
@@ -43,49 +79,111 @@ To add support of custom operations, see the [Add Custom OpenVINO Operations](..
 
 To build an `ov::Model` instance from `opset8` operations, include the following files:
 
```

Comment:
the text above says about operation sets, but we already have dedicated section.